src/test/testType.ts

```diff
@@ -34,6 +34,9 @@ export class TestTypeImpl {
   constructor(fixtures: (FixturesWithLocation | DeclaredFixtures)[]) {
     this.fixtures = fixtures;
 
+    if (process.env.JEST_WORKER_ID)
+      throw new Error(`Playwright Test needs to be invoked via 'npx playwright test' and excluded from Jest.`);
```

Comment:
Can we add something to the docs and point from here? This error does not tell me how to fix the issue.

Reply:
should be better now.